src/FilterLists.Web/ClientApp/modules/home/components/softwareIcon/SoftwareIcon.tsx

```diff
@@ -67,5 +70,8 @@ const icons: { [id: number]: IIcon; } = {
     22: { image: img22, imageTitle: "Diversion" },
     23: { image: img23, imageTitle: "dnsmasq" },
     24: { image: img24, imageTitle: "Slimjet" },
-    25: { image: img25, imageTitle: "uMatrix" }
+    25: { image: img25, imageTitle: "uMatrix" },
+    26: { image: img24, imageTitle: "Blokada" },
```

Comment:
`img24` seems to be repeated several times. Probably just a copy/paste error.

Reply:
Ah. I had forgot to change the numbers after copying and pasting an existing line. Well caught. Should be fixed now.